Runtime/src/index.ts

```diff
@@ -370,6 +396,16 @@ export class SwiftRuntime {
               const constructor = this.heap.referenceHeap(constructor_ref)
               return obj instanceof constructor
             },
+            swjs_create_typed_array: (
+                kind: JavaScriptTypedArrayKind,
+                elementsPtr: pointer, length: number,
+                result_obj: pointer
+            ) => {
+                const ArrayType: TypedArray = this.heap.referenceHeap(0)[JavaScriptTypedArrayKind[kind] + 'Array']
```

Comment:
Please don't abuse address `0` just to access global context.
It would be better to store global or window as a member of SwiftRuntime and access it directly 👍

Reply:
I created a `globalVariable` variable as a replacement. Is that a good name or should I go with something else?